explainer-client/src/main/scala/ExplainEditorJS.scala

```diff
@@ -87,8 +89,8 @@ object ExplainEditorJS {
     )
   }
 
-  def statusBarText(explainer: ExplainerItem) = {
-    val isDraftState =  !explainer.live.isDefined || explainer.draft.title!=explainer.live.get.title || explainer.draft.body!=explainer.live.get.body
+  def statusBarText(explainer: CsAtom) = {
+    val isDraftState =  true// !explainer.live.isDefined || explainer.draft.title!=explainer.live.get.title || explainer.draft.body!=explainer.live.get.body

```

Comment:
This explains why the toolbar draft state indicator remains visible after one clicks on [publish].